.circleci/run_ci.sh

```diff
@@ -0,0 +1,18 @@
+#! /bin/bash
```

Comment:
this is the whole point of the merge (adding support for the Circle-CI testing service on Linux)